CODEOWNERS

```
@@ -35,6 +35,9 @@ k8s/**/common/neuvector/fluentbit-log.yaml @hmcts/platform-engineering
 k8s/**/common/neuvector/nv-storage-secret.yaml @hmcts/platform-engineering
 k8s/**/common/neuvector/identity.yaml @hmcts/platform-engineering
 
+k8s/**/common/rpe/send-letter-identity.yaml @hmcts/bulk-scan
+k8s/**/common/rpe/send-letter-service.yaml @hmcts/bulk-scan
```

should be aat until approved by secops
```suggestion
k8s/aat/common/rpe/send-letter-service.yaml @hmcts/bulk-scan
```

---

changed